rfd/0026/README.md

```diff
@@ -1274,38 +1288,32 @@ an appropriate error message when used on shared volumes zones.
 Machines acting as shared volumes' storage zones will have the value `nfsserver`
 for their `smartdc_role` property. To know of this new `smartdc_role` value is used by CloudAPI to prevent users from performing operations on these storage VMs, refer to
 
-#### New internal `required_nfs_volumes` property on VM objects
+#### New `volumes` property on VM objects
 
-A VM object that represents a Docker container mounting a shared volumes will
-store a reference to that volume in a new _indexable_ property named
-`required_nfs_volumes`.
+A VM object that represents a container mounting a shared volumes will store a
+reference to that volume in a new _indexable_ property named `volumes`.
 
-This will allow VOLAPI to perform an indexed search on VMAPI to determine what
-uses a given volume when handling operations that depends on a volume's usage,
-e.g a `DeleteVolume` request.
+This property will also be exposed both at VMAPI and CloudAPI where it can be
```

Comment:
The comment above means we probbaly need a new section that documents the changes that would need to be made to VMAPI's `CreateVm` endpoint.